src/routes/+layout.svelte

```diff
@@ -106,8 +106,42 @@
             }
         }
     }
+
+    onMount(() => {
+        const reoIdentity = {
+            username: $user.email,
+            type: $user.password ? 'email' : 'github',
+            firstname: $user.name.split(' ')[0],
+            lastname: $user.name.split(' ')[1]
```

Comment:
What if name is null? What if they only have a 1 word name?

What happens if the user isn't logged in already?